src/config/src/config.rs

```diff
@@ -714,6 +714,12 @@ pub struct Limit {
     pub sql_min_db_connections: u32,
     #[env_config(name = "ZO_META_CONNECTION_POOL_MAX_SIZE", default = 0)] // number of connections
     pub sql_max_db_connections: u32,
+    // max time of transaction will retry
+    #[env_config(name = "ZO_META_TRANSACTION_RETRIES", default = 10)]
```

Comment:
Should we use the help?  :innocent: 

```suggestion
    #[env_config(name = "ZO_META_TRANSACTION_RETRIES", default = 10, help = "max time of transaction will retry")]
```

Reply:
Updated